docs/assemblies/custom_resources.adoc

```diff
@@ -547,22 +547,22 @@ OpenStackDataPlaneNodeSetStatus defines the observed state of OpenStackDataPlane
 | map[string]condition.Conditions
 | false
 
-| DNSClusterAddresses
+| dnsClusterAddresses
```

Comment:
You probably have to rebase on latest main https://github.com/openstack-k8s-operators/dataplane-operator/pull/649/files

Reply:
Yeah, was missing that commit. Fixed now.